src/types/modules/material-top-tabs.d.ts

```diff
@@ -0,0 +1,6 @@
+import {useTabAnimation} from '@react-navigation/material-top-tabs/src/utils/useTabAnimation';
```

Comment:
Can you help me understand the purpose of this file and why it's necessary?

Reply:
Sure! So this function was not originally in the library, afaik it was added with a patch, and it was good enough when we used JS, but once migrated to TypeScript it wasn't good enough anymore. I tried a couple solutions and this is the only one I found that works fully, importing it directly from the file and augmenting the types. We discussed it in our TS team and this is what we agreed on. Hope that answers your question!

Reply:
OK, thanks! That's good context. Would you mind adding a code comment to this file which explains that?

Reply:
Sure thing, done!